ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/Cli.scala

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.benchtool
+
+import scopt.{OParser, Read}
+
+object Cli {
+  def config(args: Array[String]): Option[Config] =
+    OParser.parse(parser, args, defaultConfig)
+
+  private val parser = {
+    val builder = OParser.builder[Config]
```

Comment:
More a question than a complaint: why have you decided to use OParser.builder rather than derive from OptionParser?

Reply:
The `OParser` is the functional alternative for parsing that `scopt` provides, so I decided to pick this one.

Reply:
Oooh, fancy. Are you going to migrate the other usages? I definitely prefer this one.

Reply:
Do you mean in other components like `ledger-api-test-tool`? I haven't planned that but can do if you think it's worth changing :)